src/Movies/Movies.js

```diff
@@ -3,9 +3,10 @@ import Poster from '../Poster/Poster.js'
 import './Movies.css'
 
 const Movies = (props) => {
+    // console.log(props)
     const allMovies = props.movieData.movies.map(movie => {
         return (
-            <Poster image={movie.poster_path} key={movie.id} displayMovieDetails={props.displayMovieDetails}/>
+            <Poster image={movie.poster_path} key={movie.id} id={movie.id} displayMovieDetails={props.displayMovieDetails}/>
```

Comment:
If you're storing the ID for later use, it's better to use a data-id attribute instead of just the id attribute